clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -115,6 +117,31 @@ public void initialize(URI name, Configuration conf) throws IOException {
             });
     }
 
+    private StorageConfig getStorageConfig(URI uri) throws IOException, ApiException {
+        Path path = new Path(uri);
+        ObjectLocation objectLoc = pathToObjectLocation(path);
+
+        Repository repository = lfsClient.getRepositoriesApi()
+                .getRepository(objectLoc.getRepository())
+                .execute();
+        String storageID = repository.getStorageId();
+        Config cfg = lfsClient.getConfigApi().getConfig().execute();
+        StorageConfig storageConfig = new StorageConfig();
```

Comment:
I don't understand - why create a blank StorageConfig?  If there is a bug and nothing is found, do you really want a _blank_ StorageConfig???
```suggestion
        StorageConfig storageConfig;
```